recipe/migrations/libxml2212.yaml

```diff
@@ -3,5 +3,5 @@ __migrator:
   kind: version
   migration_number: 1
 libxml2:
-- '2.12'
+- 2
```

Comment:
Let's keep this file for a couple of days and delete later.

Reply:
ah ok. fine by me.